### ISSUE TITLE
"Undefined but requested parser: influx" in the latest version

### ISSUE THREAD

**Issue by fudge01010** [x70b1/telegraf-edgeos#8]
Looks like a parser was potentially removed in a later version - tried `v1.25.1-2`, and when testing my config, received the following:

> E! error loading config file /etc/telegraf/telegraf.conf: error parsing exec, adding parser failed: undefined but requested parser: influx

Rolling back to `1.23.2-1` seems to resolve the issue, and my config parses fine.

Config is below:
```
 [agent]
   hostname = "telegraf_erx"

   interval = "1m"

   debug = false
   quiet = false
   logtarget = "stderr"

[[outputs.influxdb_v2]]
   urls = ["xxx"]
   organization = "xxx"
   bucket = "xxx"
   token = "xxx"

 [[inputs.cpu]]
   percpu = false
   report_active = true

 [[inputs.system]]

 [[inputs.mem]]

 [[inputs.net]]

 [[inputs.exec]]
   command = "sh /usr/lib/telegraf/edgeos.sh --interfaces --power --temperature"
   data_format = "influx"

 [[inputs.exec]]
   command = "sh /usr/lib/telegraf/edgeos.sh --firmware"
   data_format = "influx"

   interval = "1d"
```

**Comment by fudge01010** [comment]
Forgot to say - using an Edgerouter X, so `mipsel` package / build.

**Comment by x70b1** [comment]
Thanks for the report. I have found the issue and will build a new release later.

**Comment by x70b1** [comment]
Its fixed in the latest release `1.25.1-3`. 

Maybe you have an issue with the `1d` option. Then try `24h`.
